tests/41end-to-end-keys/07-backup.pl

```diff
@@ -0,0 +1,387 @@
+my $fixture = local_user_fixture();
+
+my $current_version; # FIXME: is there a better way of passing the backup version between tests?
+
+test "Can create backup version",
+   requires => [ $fixture ],
+
+   proves => [qw( can_create_backup_version )],
+
+   do => sub {
+      my ( $user ) = @_;
+
+      do_request_json_for( $user,
+         method  => "POST",
+         uri     => "/unstable/room_keys/version",
+         content => {
+            algorithm => "m.megolm_backup.v1",
+            auth_data => "anopaquestring",
+         }
```

Comment:
I think its going to be beneficial if we create `matrix_create_backup_version` etc functions. There's already a fair bit of duplication and it'll make it easier to write more backup tests.